examples/pylab_examples/cursor_demo.py

```diff
@@ -69,14 +68,14 @@ def mouse_move(self, event):
         print('x=%1.2f, y=%1.2f' % (x, y))
         draw()
 
-t = arange(0.0, 1.0, 0.01)
-s = sin(2*2*pi*t)
-fig, ax = subplots()
+t = np.arange(0.0, 1.0, 0.01)
+s = np.sin(2*2*np.pi*t)
+fig, ax = plt.subplots()
 
-cursor = Cursor(ax)
-#cursor = SnaptoCursor(ax, t, s)
-connect('motion_notify_event', cursor.mouse_move)
+# cursor = Cursor(ax)
```

Comment:
Don't use a space after a hash when commenting code in the examples. This convention is used in all examples, which I tried to follow in my pep8 PRs from last year.